save-demo/src/main/kotlin/com/saveourtool/save/demo/controller/DependencyController.kt

```diff
@@ -122,4 +125,16 @@ class DependencyController(
         "Could not find demo for $organizationName/$projectName."
     }
         .flatMapMany { dependencyStorage.archive(it.organizationName, it.projectName, version) }
+
+    /**
+     * @return save-demo-agent.kexe as [Resource] wrapped into [Mono]
+     */
+    @GetMapping("/download-agent", produces = [MediaType.APPLICATION_OCTET_STREAM_VALUE])
+    fun downloadSaveAgent(): Mono<out Resource> = run {
+        val executable = "save-demo-agent.kexe"
```

Comment:
may be use this variable as argument? in that way we can probably reuse this logic for `save-agent` too
if `save-agent` have something similar

Reply:
or at least can be reused as `SAVE_DEMO_AGENT_EXECUTABLE_NAME`

Reply:
Probably the most of this logic will be refactored in #1867 so I guess it should be done after it is merged

Reply:
Ok, add some `todo` in code, then

Reply:
Will do it

Reply:
Done